README.md

```diff
@@ -1,14 +1,10 @@
 # netatmo-weather-adapter
 Netatmo Weather Station adapter for the [Mozilla IoT gateway](https://iot.mozilla.org).
 
-Bridges the Netatmo REST API to the gateway so it can read data from Netatmo Weather Stations (and Health Coaches).
+Bridges the Netatmo REST API (via netatmo.com, not the local Netatmo device) to the gateway so it can read data from Netatmo Weather Stations (and Health Coaches).
 
 ## Usage
-Normally you will just want to install this from the add-ons list provided by
-the gateway.
+In order for your gateway to read data for your Netatmo Weather Station, you need to follow the Netatmo Weather Station setup instructions provided by Netatmo. Then install and configure this from the add-ons list provided by the gateway (Settings -> Add-ons -> Netatmo Weather Station).
```

Comment:
While I understand what you're going for, I don't think the order and information is quite right. I think it should still point out first that this is a Mozilla WebThing gateway adapter and should be installed there. Setting up the Netatmo devices honestly belongs to the setup part. So all in all, I don't think this section should have any changes.